NAMESPACE

```diff
@@ -3,5 +3,9 @@ importFrom(utils, "untar", "unzip", "contrib.url")
 export(addRepo,
        initRepo,
        insertPackage,
+       insertPackages,
+       getRepoInfo,
        pruneRepo,
-       archivePackages)
+       pruneRepoForAllRversions,
+       archivePackages,
+       archivePackagesForAllRversions)
```

Comment:
I really want to be done with the PR but dropping new functions into the API without first discussing it is ... a little on the strong-willed side too.  Here I am ok with your changes; the process was a little on the heavy side.

Reply:
Sorry for that. I didn't want to interfere with the expectation of current users, that more than R version would be targeted. I couldn't think of an alternative.

Reply:
I think the design is entirely defensible and possibly even good :) (I don't really use these archiver and pruner functions in _my_ workflow so I am mostly :man_shrugging:  here) but the _process of dropping them unannounced_ is not one I prefer in my repos.  That was all.

Reply:
Totally agree, However, I didn't drop anything, just slotted the additional functions in there at there appropriate places.